guess_the_words/function.py

```diff
@@ -0,0 +1,161 @@
+import json
+import string
+import random
+import datetime
+
+
+def create_list():
+    with open('words.txt', 'r') as file:
+        list_words = file.readlines()
+        for posit, word in enumerate(list_words):
+            list_words[posit] = word.strip()
+    return list_words
+
+
+def val_count(player_letters: list) -> list:
```

Comment:
```suggestion
def val_count(player_letters: list) -> bool:
```

Reply:
I understood why bool - this is result return function. But i delletet this function and wrote code instead use fucntions val_count and  val_alph